### PR TITLE
[NEW] Install and Configure ownCloud

### PR DESCRIPTION
**Pull request by Guaris**

### CODE REVIEW

File: docs/applications/owncloud.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Login to your MySQL database, and enter your root password: 
+
+    mysql -p -u root
+
+Create a new database for ownCloud:
+
+    create database ownCloud;
+	create user ownCloud@localhost
+
+Assign the user to the database:
```
Comment (GuessWhoSamFoo) [inline docs/applications/owncloud.md]: Not sure how this is any better than just using `root` if creating a new user with all privileges and no password.

File: docs/applications/owncloud.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+The `clamav` package starts a daemon on your system. 
+
+Enable the antivirus app in ownCloud:
+
+![antivirus](/docs/assets/ownCloud/antivirus.png)
+
+Configure your antivirus mode, in ownCloud, to reflect the changes to your system:
+
+![socket](/docs/assets/ownCloud/owncloudsocket.png)
```
Comment (GuessWhoSamFoo) [inline docs/applications/owncloud.md]: This picture shows the DigitalOcean guide for installing Java using `apt-get`

File: docs/applications/owncloud.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+	+--------------------+
+	1 row in set (0.00 sec)
+
+
+
+### Create an Admin Account
+
+After ownCloud is installed, and MySQL is configured, point your browser to `ip_address_or_domain/owncloud`. 
+
+![owncloudlogin](/docs/assets/ownCloud/login.png)
```
Comment (GuessWhoSamFoo) [inline docs/applications/owncloud.md]: Missing step to select MySQL/MariaDB instead of SQLite under `Storage and database`

**Comment by Guaris** [comment]
@GuessWhoSamFoo Revisions made.

**Pull request merged**